components/adblock_rust_ffi/src/lib.rs

```diff
@@ -343,6 +343,22 @@ pub unsafe extern "C" fn filter_list_metadata_title(
     }
 }
 
+/// Returns the amount of time this filter list should be considered valid for,
+/// in hours. Defaults to 168 (i.e. 7 days) if unspecified by the
+/// `FilterListMetadata`.
+#[no_mangle]
+pub unsafe extern "C" fn filter_list_metadata_expires(metadata: *const FilterListMetadata) -> u16 {
+    use adblock::lists::ExpiresInterval;
+
+    const DEFAULT_EXPIRES_HOURS: u16 = 7 * 24;
```

Comment:
It's the same as `kSubscriptionDefaultExpiresHours`.
Could we avoid duplication of this const somehow?

Reply:
I suppose this could be exported as part of the FFI, I'll give it a try

Reply:
exporting worked without any issues; deduplicated everywhere